lib/serialize-value.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var destroyCircular = require('destroy-circular');
+
+// Make a value ready for JSON.stringify() / process.send()
+
+module.exports = function serializeValue(value) {
+	if (typeof value === 'object') {
+		return destroyCircular(value);
+	}
+	if (typeof value === 'function') {
+		// JSON.stringify discards functions
```

Comment:
Does not work in node `5.0.0`. Empty strings all around.
Works in Babel instrumented code though.